quadcopter_brain/src/quadcopter_brain/landing_site.py

```diff
@@ -70,6 +71,29 @@ def lat_lon(self, copter):
                                             absolute_site[0][0],
                                             absolute_site[1][0])
 
+    def get_average_lat_long(self, copter, total_time=5.0, time_step=0.1):
+        '''
+        Waits for 'time' seconds and samples landing site position
+        every 'time_step' seconds. Returns the average gps position
+        over that time period. Returns None if fiducial never seen
+        '''
+        landing_site_lat = []
+        landing_site_long = []
+        time_limit = datetime.timedelta(seconds=total_time)
+        time_end = datetime.datetime.now() + time_limit
```

Comment:
doesn't this give a datetime object? has it been proven to work? trying to do something similar in an interactive python interpreter gave me an error